components/brave_ads/core/internal/ad_units/new_tab_page_ad/new_tab_page_ad_handler.cc

```diff
@@ -16,10 +16,10 @@
 #include "brave/components/brave_ads/core/internal/targeting/geographical/subdivision/subdivision_targeting.h"
 #include "brave/components/brave_ads/core/internal/user_engagement/ad_events/new_tab_page_ads/new_tab_page_ad_event_handler.h"
 #include "brave/components/brave_ads/core/internal/user_engagement/site_visit/site_visit.h"
+#include "brave/components/brave_ads/core/mojom/brave_ads.mojom-shared.h"
```

Comment:
not needed as `brave_ads.mojom.h` is already included below.

Reply:
Fixed, thanks